Cubical/Foundations/Embedding.agda

```diff
@@ -0,0 +1,81 @@
+{-# OPTIONS --cubical --safe #-}
```

Comment:
What is the relationship between the content of this file and https://github.com/agda/cubical/blob/master/Cubical/Foundations/PathSplitEquiv.agda ?

Reply:
I'm not really sure. It looks like a `PathSplitEquiv` is similar to an equivalence. But embeddings are a notion similar to injective maps that are properly behaved for higher hlevel types.

Reply:
Ok, it seemed to me that the results you proved were similar... But I was also very tired when I looked at it so I might have been mistaken